src/Fantomas/CodePrinter.fs

```diff
@@ -1491,14 +1494,14 @@ and genInfixApps astContext (hasNewLine:bool) synExprs (ctx:Context) =
 /// Use in indexed set and get only
 and genIndexers astContext node =
     match node with
-    | Indexer(Pair(IndexedVar eo1, IndexedVar eo2)) :: es ->
+    | Indexer(Pair((IndexedVar eo1, e1FromEnd),(IndexedVar eo2, e2FromEnd))) :: es ->
```

Comment:
I'm not _using_ the from-end-ed-ness here, but I'm fairly certain it would need to result in a change to the indexer syntax